test/test-pexprs-madlibs.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+var test = require('tape');
+
+var makeGrammar = require('./testUtil').makeGrammar;
+
+// --------------------------------------------------------------------
+// Tests
+// --------------------------------------------------------------------
+
+// test('PExpr');
+//
+test('Alt', function(t) {
+  var g = makeGrammar([
+    'G {',
+    '  alt = "a" | "b"',
+    '}'
+  ]);
+
+  var random = Math.random;
+
+  Math.random = function() { return 0.6; };
+  var example = g.rules.alt.body.generateExample(g, {}, false, []);
+  t.equal(example.example, 'b');
```

Comment:
Hmmm, maybe the `example` property should be named `value` instead? It looks a bit weird to write code like `example.example`.